### PR TITLE
Make header section different from master section

### PR DESCRIPTION
**Pull request by BonkeGcobo**
Only a few changes have been made to the code, so I can be able submit last weeks code,

### CODE REVIEW

File: css/myStyle.css [hunk 1]
```diff
@@ -43,7 +43,7 @@ main {
   font-size: 20px;
```
Comment (carlos-ssh) [inline css/myStyle.css:45]: - Import styles from your CSS file( `` @import url('https://fonts.googleapis.com/css2?family=Poppins'); `` )

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!
Your project it's nice, just fix the requested changes below


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @BonkeGcobo 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:

- [x] To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### You have the descriptive PR:heavy_check_mark:
   * #### Your project looks awesome.:heavy_check_mark:
   
Changes required :writing_hand:  

    - You are missing some files in the linters configurations (.hintrc and .sytlelintrc.json), please refer to the [template](https://github.com/microverseinc/linters-config) again for more.
    
    -  Kindly, add new folder in the root folders to keep the images.
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by BonkeGcobo** [comment]
Hey @Emmyn5600, can I have clarity on the .hintric and stylelintric.json,, I added the linters as I should, plus this is a project that I did last week and it was approved.

File: Portfolio.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
       <header class="header-container">
```
Comment (SajjadAhmad14) [inline Portfolio.html:18]: Please use this linter config. The project is missing `.stylelintrc.json` and `.hintrc` files.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio
+# Microverse
 
 > Create Professional Portifolio
```
Comment (SajjadAhmad14) [inline README.md:5]: [ OPTIONAL ] README should have a description of what this project is about and its features, and instructions on how to set the project up locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bonke Gcobo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is used.
- [ ] Linter is set up perfectly.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by SpaYco — APPROVED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

only `.github/workflows/linters.yml` needs to be present for linters to work on GitHub, some CRs may not realize that, I'll make sure to let everyone know, sorry for the inconvenience!

As this was the only reason that your project was not approved it will be approved as soon as possible.

Keep rocking 🚀!

**Pull request merged**